.github/workflows/codeql-analysis.yml

```diff
@@ -13,7 +13,6 @@ name: "CodeQL"
 
 on:
   push:
-    branches: [ main, * ]
```

Comment:
is this going to run on push to all branches?

Reply:
Yes! The previous syntax was wrong.